test/README.md

```diff
@@ -1,4 +1,8 @@
-# Tests
+# Table of Contents
+* [test directories](#tests)
+* [common module API](#commonjs)
```

Comment:
Sounds good. I'll change the headers of each section to match the link text here.